test/controllers/admin/attendee_controller_test.exs

```diff
@@ -0,0 +1,37 @@
+defmodule Syndeo.Admin.AttendeeControllerTest do
+  use Syndeo.ConnCase
+
+  alias Syndeo.Attendee
+
+  setup _ do
+    {
+      :ok,
+      conn: Phoenix.ConnTest.build_conn(),
+      id: build(:user) |> save |> Map.get(:id),
+    }
+  end
+
+  test "lists all entries on index", %{conn: conn, id: id} do
+    conn = get conn, admin_attendee_path(conn, :index, as: id)
```

Comment:
This doesn't actually test that it lists all entries, it only tests that the text "attendees" shows up which doesn't seem terribly useful.

What do you think about creating some attendees and verifying that they show up or killing the test if it's not valuable?